streamer/src/nonblocking/quic.rs

```diff
@@ -700,6 +720,33 @@ async fn packet_batch_sender(
     }
 }
 
+fn track_streamer_fetch_packet_performance(
+    packet_batch: &PacketBatch,
+    packet_perf_measure: &mut HashMap<[u8; 64], Instant>,
+    stats: Arc<StreamStats>,
```

Comment:
we can take a reference ot `&StreamStats`, then we don't need to clone this at call-site. Am I missing something in the fn that requires us to own this?

Reply:
Oh yes.

Reply:
Done